src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ActionPlanServiceTest.kt

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.context.transaction.TestTransaction
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.aValidPrisonNumber
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.anotherValidPrisonNumber
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.aValidActionPlan
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.aValidActionPlanSummary
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.service.ActionPlanService
+
+@Deprecated("A temporary test until we have integration tests which use the relevant controller endpoints")
```

Comment:
This is removed in my next PR: https://github.com/ministryofjustice/hmpps-education-and-work-plan-api/pull/81